### PR TITLE
Boyardee-B update

### PR DESCRIPTION
**Pull request by ToasterBan**
## About The Pull Request
Updates the Boyardee-B, list of all changes will be done later

the main intent of these changes is to make the boyardee-B a more combat capable but still restaurant capable ship

- removed the vault, replaced with a botany/coldroom area
- the carp and punpuns have been removed, replaced by Bitey the alligator
- added the syndicate waiter suit and respective job which is the assistant role of the Boyardee-B
- AAC starts unlocked
- minor tile changes so carpets look a bit better
- Suit Storage's are now equipped with syndie masks and suits (not hardsuits)
- Added some extra engines and more fuel for said engines allowing it to be better for combat
- regular pacmans replaced with superpacmans
- added a 3 use emag, use it wisely
- added dice and a roulette machine to add to the casino aesthetic 
- fixed some small lighting issues
- 

## Why It's Good For The Game
the normal version got an update so why not give its edgier brother a little love?

## Changelog
:cl:
add: Boyardee-B class updates
/:cl:

### CODE REVIEW

**Comment by Calyxman** [comment]
Pretty cool, what about adding a Clown job to it? Keep the safe, put it somewhere under the floor though.

**Comment by ToasterBan** [comment]
a clown role could be interesting but the main idea if the ship is that it's a restaurant-casino and I don't know how clowns will fit in

**Comment by Calyxman** [comment]
> a clown role could be interesting but the main idea if the ship is that it's a restaurant-casino and I don't know how clowns will fit in

the clown encourages you to gamble.
in all seriousness i feel like not too many players gamble, let alone interact with boyardees frequently. it makes sense for an entertainment vessel to have entertainers (although the clown would be a little hard to make space for)
also holy shit we need this speedmerged

**Comment by Calyxman** [comment]
finally tmerged. is the exclusion of a nutrient vendor/botany tools intentional?

**Comment by ToasterBan** [comment]
> finally tmerged. is the exclusion of a nutrient vendor/botany tools intentional?

yes, you are trading some restaurant-ness for weapons and a faster ship

File: _maps/shuttles/shiptest/bar_ship_b.dmm [hunk 1]
```diff
@@ -2308,28 +2323,40 @@
...
-/obj/structure/bed/dogbed/cayenne,
-/mob/living/simple_animal/hostile/carp/cayenne,
+/obj/structure/bed/dogbed{
+	desc = "A comfy-looking gator bed. You can even strap your gator in, in case the gator turns off.";
+	name = "Bitey's Bed"
+	},
+/mob/living/simple_animal/hostile/retaliate/gator/steppy{
+	attack_verb_continuous = "tries to bite";
+	attack_verb_simple = "tries to bite";
+	desc = "Bitey the gator! it appears to have lost its teeth";
```
Comment (ToasterBan) [inline _maps/shuttles/shiptest/bar_ship_b.dmm]:
```suggestion
	desc = "Bitey the gator! It appears to have lost its teeth";
```
spelloing

**Comment by Calyxman** [comment]
> > finally tmerged. is the exclusion of a nutrient vendor/botany tools intentional?
> 
> yes, you are trading some restaurant-ness for weapons and a faster ship

a little harder to do botany without, like, a cultivator. iirc you can make pest spray in the biogen and nutrients in the biogen as well, but you can't make cultivators.

**Comment by ToasterBan** [comment]
alright added some botany tools, botany should be doable while the "restaurant is weaker" element still taking part

**Comment by Calyxman** [comment]
> alright added some botany tools, botany should be doable while the "restaurant is weaker" element still taking part

Based ship dev?! Listens to criticisms? Thank you man

**Comment by ToasterBan** [comment]
> > alright added some botany tools, botany should be doable while the "restaurant is weaker" element still taking part
> 
> Based ship dev?! Listens to criticisms? Thank you man

The entire point of this pr is to improve the ship for everyone

I am planning on updating other ships but I am less accustomed to them so I highly reccomend going on the forums and throwing your suggestions in the ship's thread in here: https://forums.shiptest.net/c/coder-hell/map-review/15

**Pull request merged**